docs/user-guide/station-authentication-modes.md

```diff
@@ -71,6 +71,16 @@ Supported values for 'CLIENT_CERTIFICATE_MODE' environment variables are:
 - `1` or `AllowCertificate`: client certificate will be requested; however, authentication will not fail if a certificate is not provided by the client
 - `2` or `RequireCertificate`: client certificate will be requested, and the client must provide a valid certificate for authentication to succeed
 
+When a client certificate is used, two verifications are done on the 'LoRaWanNetworkSrvModule' side:
+
+- The certificate chain is validated.  
```

Comment:
very nit: you say "two verifications are done", so you could use a numbered list